homeassistant/components/remember_the_milk.py

```diff
@@ -0,0 +1,249 @@
+"""Component to interact with Remember The Milk.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/remember_the_milk/
+
+Minimum viable product, it currently only support creating new tasks in your
+Remember The Milk (https://www.rememberthemilk.com/) account.
+
+This product uses the Remember The Milk API but is not endorsed or certified
+by Remember The Milk.
+"""
+import logging
+import os
+import json
+import voluptuous as vol
+
+from homeassistant.config import load_yaml_config_file
+from homeassistant.const import (CONF_API_KEY, STATE_OK, CONF_TOKEN, CONF_NAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+
+REQUIREMENTS = ['RtmAPI==0.7.0', 'httplib2==0.10.3']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'remember_the_milk'
+DEFAULT_NAME = DOMAIN
+GROUP_NAME_RTM = 'remember the milk accounts'
+
+CONF_SHARED_SECRET = 'shared_secret'
+
+RTM_SCHEMA = vol.Schema({
+    vol.Required(CONF_API_KEY): cv.string,
+    vol.Required(CONF_SHARED_SECRET): cv.string,
+})
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: {
+        cv.string: RTM_SCHEMA
+    }
+}, extra=vol.ALLOW_EXTRA)
+
+CONFIG_FILE_NAME = 'remember_the_milk.conf'
```

Comment:
Please prefix it with a `.` because it's not supposed to be editted by users.

Reply:
OK